tests/book.test.js

```diff
@@ -0,0 +1,139 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const { Book } = require('../src/models');
+const app = require('../src/app');
+
+describe('/books', () => {
+  before(async () => Book.sequelize.sync());
+
+  afterEach(async () => {
+    await Book.destroy({ where: {} });
+  });
+
+  describe('with no records in the database', () => {
```

Comment:
Your indentation is way off here. Please make sure your formatting is uniform thorughout